lib/gpu/Install.py

```diff
@@ -23,97 +30,75 @@
   and to copy a Makefile.lammps.esuffix -> Makefile.lammps
 optionally copies Makefile.auto to a new Makefile.osuffix
 
-  -m = use Makefile.machine as starting point, copy to Makefile.auto
-       default machine = linux
-       default for -h, -a, -p, -e settings are those in -m Makefile
-  -h = set CUDA_HOME variable in Makefile.auto to hdir
-       hdir = path to NVIDIA Cuda software, e.g. /usr/local/cuda
-  -a = set CUDA_ARCH variable in Makefile.auto to arch
-       use arch = sm_20 for Fermi (C2050/C2070, deprecated as of CUDA 8.0)
-                        or GeForce GTX 580 or similar
-       use arch = sm_30 for Kepler (K10)
-       use arch = sm_35 for Kepler (K40) or GeForce GTX Titan or similar
-       use arch = sm_37 for Kepler (dual K80)
-       use arch = sm_60 for Pascal (P100)
-       use arch = sm_70 for Volta
-  -p = set CUDA_PRECISION variable in Makefile.auto to precision
-       use precision = double or mixed or single
-  -e = set EXTRAMAKE variable in Makefile.auto to Makefile.lammps.esuffix
-  -b = make the GPU library using Makefile.auto
-       first performs a "make clean"
-       then produces libgpu.a if successful
-       also copies EXTRAMAKE file -> Makefile.lammps
-         -e can set which Makefile.lammps.esuffix file is copied
-  -o = copy final Makefile.auto to Makefile.osuffix
-
 Examples:
 
 make lib-gpu args="-b"      # build GPU lib with default Makefile.linux
```

Comment:
Some help info was lost here, i.e. `use arch = sm_35 for Kepler (K40) or GeForce GTX Titan or similar`. Is that covered somewhere else?

Reply:
This help info was very limited so most people will have to look up what to use anyway, and the reference to GTX Titan is misleading, since there are now many Titan variants with different labels and architectures. Overall, it would be a major PITA to maintain a reasonably meaningful list in this place. I think a better help would be to find a suitable table on the web somewhere, that is also maintained and covers the whole history (so nvidia's webpages don't cut it). how about [this page on wikipedia]( https://en.wikipedia.org/wiki/CUDA#GPUs_supported)

Reply:
The README file in `lib/gpu` is also in dire need of an update/refresh/cleanup. Since the manual has become much more comprehensive and complete on these issues, it is probably best to reduce the information here to the minimum and refer to the corresponding sections in the manual instead.